### PR TITLE
Organized chaos

### PR DESCRIPTION
**Pull request by mistysloth**
Easter egg and the sudden appearance of misty.

### CODE REVIEW

File: features/features.json [hunk 1]
```diff
@@ -1108,5 +1108,14 @@
...
     "type": ["Website", "Theme"]
+  },
+  {
+    "title": "Organized Chaos",
+    "description": "hmm. i should enable this.",
+    "credits": ["Misty"],
+    "urls": ["https://github.com/mistysloth/"],
+    "file": "organized-chaos",
+    "tags": ["Easter Egg"],
+    "type": ["Website", "Theme", "Easter Egg"]
```
Comment (rgantzos) [inline features/features.json:1119]: This should be using Feature v2.

**Review by rgantzos — CHANGES_REQUESTED** [review]

File: features/organized-chaos.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+setTimeout(() => {
+  javascript:(function(){['', '-ms-', '-webkit-', '-o-', '-moz-'].map(function(prefix){document.body.style[prefix + 'transform'] = 'rotate(180deg)';});}());
```
Comment (Samq64) [inline features/organized-chaos.js:2]: This has been bugging me. Why is there a JavaScript URI inside a JavaScript file?

Comment (callumjt) [reply features/organized-chaos.js:2]: Wtf even is that feature

Comment (callumjt) [reply features/organized-chaos.js:2]: > Wtf even is that feature

Oh wait this pr is for that feature 💀

Comment (RedGuy12) [reply features/organized-chaos.js:2]: is that not a syntax error

Comment (Samq64) [reply features/organized-chaos.js:2]: I think I'll regret testing this if it does work...

Comment (Samq64) [inline features/organized-chaos.js:2]:
```suggestion
  document.body.style.transform = 'rotate(180deg)';
```
Or even better, but use a CSS file. Tested on Chromium. I couldn't get the extension to load on Firefox.

Comment (callumjt) [reply features/organized-chaos.js:2]: Ste isn't fully supported on Firefox yet

Comment (rgantzos) [reply features/organized-chaos.js:2]: > is that not a syntax error

It's technically not. I'm assuming this code was found somewhere else.

Comment (mistysloth) [reply features/organized-chaos.js:2]: the code is from my former cs teacher. it's not mine.

**Pull request merged**